### DISCUSSION TITLE
Script inside component loading only on page refresh.

### DISCUSSION THREAD

**Discussion by Idhavalkacha** [livewire/livewire#7299] [1. Help]
Hey there! 
I'm working with multi step forms. I've total 3 forms which I'm showing conditionally on $currentStep.

Now problem here is I want to show payment button of razorpay and this is using this script tag to show the button, which code I've mentioned below.
But the script is loading only once when refresh the page, but when I go to second form and come back to first one then the script isn't loading. Also I'm loading this script conditionally (i want to show payment button for form1 only).

Is there any way to solve this script rendering issue?

```
@if ($currentStep == 1)
        <form action="{{ route('payment.store') }}" method="POST">
            @csrf
            <script src="https://checkout.razorpay.com/v1/checkout.js"
                    data-key="{{ env('RAZORPAY_KEY_ID') }}"
                    data-amount="{{$price}}"
                    data-buttontext="Pay {{$price}} INR"
                    data-name="{{ env('APP_NAME') }}"
                    data-description="Razorpay payment"
                    data-image="/images/logo-icon.png"
                    data-prefill.name="TEST"
                    data-prefill.email="test@gmail.com"
                    data-theme.color="#ff7529">
            </script>
        </form> 
@endif

```

[screen-capture.webm](https://github.com/livewire/livewire/assets/93464574/9060075b-5d06-4f2a-bf00-3a19a4deea5d)

Thanks!

**Comment by freshface** [comment]
Add a wire:ignore to the form?

**Reply by Idhavalkacha** [reply]
already tried! Not working.
